src/OrleansProviders/SQLServer/SqlStatisticsPublisher.cs

```diff
@@ -21,20 +21,20 @@ MIT License
 TORT OR OTHERWISE, ARISING FROM, OUT OF OR IN CONNECTION WITH THE SOFTWARE OR THE USE OR OTHER DEALINGS IN THE SOFTWARE.
 */
 
+using Orleans.Runtime;
+using Orleans.Runtime.Configuration;
+using Orleans.Runtime.Storage.Relational;
 using System;
 using System.Collections.Generic;
+using System.Data;
 using System.Linq;
 using System.Net;
 using System.Threading.Tasks;
-using System.Data;
-using System.Data.SqlClient;
 
-using Orleans.Runtime;
-using Orleans.Runtime.Configuration;
 
 namespace Orleans.Providers.SqlServer
 {
-    public class SqlStatisticsPublisher : IConfigurableStatisticsPublisher, IConfigurableSiloMetricsDataPublisher, IConfigurableClientMetricsDataPublisher, IProvider
+    public class SqlStatisticsPublisher: IConfigurableStatisticsPublisher, IConfigurableSiloMetricsDataPublisher, IConfigurableClientMetricsDataPublisher, IProvider
```

Comment:
This class felt a bit messy with all the ``Init`` functionality and so forth. I'm not sure if there's a better way to do logging too. Ideas gladly welcomed here.